about.html

```diff
@@ -0,0 +1,127 @@
+                            </div>
+                        </a>
+                    </li>
+                    <li class="mobile-nav-li">
+                        <a href="#" class="mobile-nav-link" id="about-link">
+                            <div class="mobile-nav-element-container">
+                                <h3 class="mobile-nav-link-title">About</h3>
+                                <img src="img/icon_next.png" alt="" class="mobile-nav-icon-next">
+                            </div>
+                        </a>
+                    </li>
+                </ul>
+            </nav>
+    
+        </header>
+        
+        <main>
+            <div class="welcome-container">
+                <h2 class="welcome-subtitle">Hello Sharing World</h2>
+                <h1 class="welcome-title">CREATIVE COMMONS GLOBAL SUMMT 2022</h1>
+                <div class="welcome-description-container-about">
+                    <p class="welcome-description-text-about">
+                      the global summit brings together the community of experts, academics and activists who comprise the creative commons affiliate network in a different country every two. <br />this year we hope to expand our invitation list including organizations and individuals.
+                    </p>
+                </div>
+                <div class="welcome-contact-container">
+                    <h4 class="welcome-address-txt">Please contact us per email for any further question about CC Global summit 2015</h4>
+                    <a id="welcome-contact-email" href="mailto:ccsummit2015@cckorea.com">ccsummit2015@cckorea.com</a>
+                </div>
+            </div>
+            <div class="cc-logo-container">
+                <h2 class="cc-logo-title">Creative Common Global Summit 2015 logo</h2>
+                <h5 class="cc-logo-desc">the logo of CC Summit 2015 was decided through the logo competition from 5th to 6th jully</h5>
+                <img class="cc-logo-img" src="img/cc_logo_black.png" alt="cc logo" />
+                
+            </div>
+            <div class="past-cc-container" id="past-cc-summit">
+                <h2 class="past-cc-title">See the past CC Global Summit</h2>
+                <h4 class="past-cc-desc">Take a look on the last two CC Global Summit that took place in Buenos Aires and in Warsaw</h4>
+                <div class="article-holder">
+                    <article class="past-cc-article cc-article-buenos">
+                        <h2 class="past-cc-article-date">2013</h2>
+                        <h4 class="past-cc-article-desc">CC Global Summit in Buenos Aires</h4>
+                    </article>
+                     <article class="past-cc-article cc-article-warsaw">
+                        <h2 class="past-cc-article-date">2011</h2>
+                        <h4 class="past-cc-article-desc">CC Global Summit in Warsaw</h4>
+                    </article>
+                </div>
```

Comment:
- NB: Please, the Section tag `<section>` defines the section of documents such as chapters, headers, footers or any other sections. Kindly make use of it in About and index HTML.
- Use this link to learn more about[ Semantic HTML ](https://www.w3schools.com/html/html5_semantic_elements.asp)